examples/ex01_basics/IncrValueModular_test.py

```diff
@@ -38,8 +38,8 @@ def construct( s ):
     s.buf1 = Wire( Bits8 )
     s.buf2 = Wire( Bits8 )
 
-    s.connect( s.in_, s.buf1 )
-    s.connect( s.out, s.buf2 )
+    connect( s.in_, s.buf1 )
```

Comment:
So the use of shorthand `//=` versus `connect` is just arbitrary? Do we have any preference of using one over another in our own tests?

Reply:
I didn't have a good answer for this ... I arbitrarily port some s.connect to connect and some to //=.